stor/obs.py

```diff
@@ -293,7 +293,10 @@ def stream_cls(self):
     def _buffer(self):
         "Cached buffer of data read from or to be written to Object Storage"
         if self.mode in ('r', 'rb'):
-            return self.stream_cls(self._path.read_object())
+            try:  # Catch possible bytes / str issues on py3k
+                return self.stream_cls(self._path.read_object())
```

Comment:
ah okay, so I was expecting that read_object always returns bytes in Python 3, so then we decode to utf-8 if you open it with 'w' or 'r' - right? I think that's how the filesystem works in Python 3?